src/nyc_trees/apps/core/models.py

```diff
@@ -88,6 +95,10 @@ def clean(self):
     def get_absolute_url(self):
         return reverse('user_detail', args=[self.username])
 
+    def completed_training(self):
```

Comment:
There are two types of training, online and field, so to keep things unambiguous, we should probably name this ``completed_online_training``